### ISSUE TITLE
Issue: Site Settings - If you add multiple alias's that are the same duplicate entries are created.

### ISSUE THREAD

**Issue by thabaum** [oqtane/oqtane.framework#3647]
### Steps to Reproduce
1. Create a site
2. Create an alias that is HTTP and one that is HTTPS for the same domain alias after the // ( https://domain.com, http://domain.com)
3. Results is two alias's available

### Expected Behavior
A message saying this alias already exists.

### Additional Context
This may also allow domain.com http://domain.com and https://domain.com and end up with 3.   i have not tested how many times and different ways I can make the same alias appear.

**Comment by sbwalker** [comment]
Alias names should not include the protocol. The validation should remove any protocol prefix which is entered.

**Comment by thabaum** [comment]
> Alias names should not include the protocol. The validation should remove any protocol prefix which is entered.

Correct, this is the current behavior.  The protocol is removed.  A duplicate is made and saved.

![image](https://github.com/oqtane/oqtane.framework/assets/13577556/e77c463f-6858-4f3b-a386-6d4cf0da6b9f)

![image](https://github.com/oqtane/oqtane.framework/assets/13577556/0073213e-d2ea-4f74-aee4-7ac401729bbc)

![image](https://github.com/oqtane/oqtane.framework/assets/13577556/81ec699e-1571-498e-923a-2e454d938c04)

![image](https://github.com/oqtane/oqtane.framework/assets/13577556/88528854-2809-4bdb-9e4f-200c2985fb60)

**Comment by thabaum** [comment]
@sbwalker dont sweat this one I have a PR to submit to handle it.
